packages/fluentui/react-northstar-fela-renderer/test/felaRenderer-test.tsx

```diff
@@ -1,20 +1,42 @@
 import { createFelaRenderer } from '@fluentui/react-northstar-fela-renderer';
 import { ICSSInJSStyle } from '@fluentui/styles';
-// @ts-ignore
-import { createSnapshot } from 'jest-react-fela';
+import { renderToString } from 'fela-tools';
+import { format } from 'prettier';
 import * as React from 'react';
-import { FelaComponent } from 'react-fela';
+import * as ReactDOM from 'react-dom';
+import { FelaComponent, RendererProvider, ThemeProvider } from 'react-fela';
 
 const felaRenderer = (createFelaRenderer() as any).getOriginalRenderer();
 
+function createSnapshot(component: JSX.Element, theme = {}) {
+  const div = document.createElement('div');
+
+  // reset renderer to have a clean setup
+  felaRenderer.clear();
+
+  ReactDOM.render(
+    <RendererProvider renderer={felaRenderer}>
+      <ThemeProvider theme={theme}>{component}</ThemeProvider>
+    </RendererProvider>,
+    div,
+  );
+
+  const css = renderToString(felaRenderer);
+  const formattedCss = format(css, { parser: 'css', useTabs: false, tabWidth: 2 });
+  // jest-react-fela used htmltojsx to format the HTML, but that no longer works with React 17
+  // due to importing a removed module. Various alternatives are available, but in this case,
+  // all the things to be serialized are simple enough that we can just use innerHTML directly.
+  return `${formattedCss}\n\n${div.innerHTML}`;
+}
```

Comment:
This is probably don't a big issue, but it might be a good idea to call `ReactDOM.unmountComponentAtNode`. In this case it will look something like below:

```suggestion
  ReactDOM.render(
    <RendererProvider renderer={felaRenderer}>
      <ThemeProvider theme={theme}>{component}</ThemeProvider>
    </RendererProvider>,
    div,
  );

  // jest-react-fela used htmltojsx to format the HTML, but that no longer works with React 17
  // due to importing a removed module. Various alternatives are available, but in this case,
  // all the things to be serialized are simple enough that we can just use innerHTML directly.
  const innerHTML = div.innerHTML;

  const css = renderToString(felaRenderer);
  const formattedCss = format(css, { parser: 'css', useTabs: false, tabWidth: 2 });

  ReactDOM.unmountComponentAtNode(div);

  return `${formattedCss}\n\n${innerHTML}`;
```